cmd/operator/service.go

```diff
@@ -83,18 +88,22 @@ func getServicesByRegionAndLabel(ctx context.Context, logger *logrus.Logger, pro
 // If the target configuration does not specify any regions, the entire list of
 // regions is retrieved from API.
 func determineRegions(ctx context.Context, logger *logrus.Logger, target *config.Target) ([]string, error) {
+	lg := logger.WithField("labelSelector", target.LabelSelector)
```

Comment:
label selector is not applicable to determining regions. so probably this log stmt shouldn't have this field